grapple/types/images.py

```diff
@@ -175,3 +197,23 @@ def resolve_image_type(self, info, **kwargs):
             return get_image_type()
 
     return Mixin
+
+
+def pillow_blur(image):
+    from PIL import ImageFilter
+
+    blurred_image = image.image.filter(ImageFilter.BLUR)
+    return PillowImage(blurred_image)
+
+
+def wand_blur(image):
+    # Wand modifies images in place so clone it first to prevent
+    # altering the original image
+    blurred_image = image.image.clone()
+    blurred_image.gaussian_blur()
+    return WandImage(blurred_image)
+
+
+# Register the operations in Willow
+willow_registry.register_operation(PillowImage, "blur", pillow_blur)
+willow_registry.register_operation(WandImage, "blur", wand_blur)
```

Comment:
suggestion: namspace this to something like `grapple_blur` in the event Willow adds its own blur operation, or site implementer for that matter